### PR TITLE
get insert kb to work in input editor

### PR DESCRIPTION
**Pull request by meganrogge**
fixes https://github.com/microsoft/vscode-copilot/issues/4797

### CODE REVIEW

File: src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatActions.ts [hunk 1]
```diff
@@ -259,7 +260,8 @@ registerActiveXtermAction({
 		when: TerminalChatContextKeys.requestActive.negate(),
 		weight: KeybindingWeight.WorkbenchContrib,
 		primary: KeyMod.Alt | KeyCode.Enter,
-	},
+		secondary: [KeyMod.CtrlCmd | KeyCode.Enter | KeyMod.Alt]
+	}],
```
Comment (meganrogge) [inline src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatActions.ts:264]:
```suggestion
	},
```

**Comment by meganrogge** [comment]
I'm still not sure how users are supposed to discover `secondary` keybindings

**Comment by meganrogge** [comment]
Perhaps we should have a second keybinding with a `when` clause like I did in the first commit on this PR and call `updateToolbar` when focus moves to/from the inline chat widget input so the hover reflects the correct keybinding to use?

**Review by Tyriar — APPROVED** [review]
This is just for those that discover the keybinding by focusing the code editor actions while their focus is in the chat input. So we will do likely what the user wants in this case instead of nothing.

**Pull request merged**